src/index.html

```diff
@@ -9,10 +10,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is linking to a `.scss` file. Browsers cannot directly interpret SCSS files. Please compile your SCSS into a CSS file and update this link to point to the compiled CSS file.